src/xrServerEntities/script_ini_file.cpp

```diff
@@ -6,13 +6,17 @@
 //	Description : Script ini file class
 ////////////////////////////////////////////////////////////////////////////
 
+#include "pch_script.h"
 #include "StdAfx.h"
 #include "script_ini_file.h"
 #include "xrScriptEngine/script_engine.hpp"
 #include "ai_space.h"
 #include "object_factory.h"
 #include "xrCore/xrDebug.h"
 
+using namespace luabind;
+using namespace luabind::policy;
```

Comment:
Since, everything was moved inside `script_ini_file_script.cpp`, please, revert this file 👍